package.json

```diff
@@ -87,12 +88,14 @@
     "react-dom": "19",
     "react-is": "^19",
     "rimraf": "^6.0.1",
-    "sanity": "^3.74.1",
+    "sanity": "^4.1.0",
     "semantic-release": "^24.2.0",
     "styled-components": "^6.1.15",
-    "typescript": "^5.1.6"
+    "typescript": "^5.1.6",
+    "vitest": "^3.2.4"
   },
   "peerDependencies": {
+    "@sanity/vision": "^4.1.0",
```

Comment:
was this intentional? seems like the only place vision is used is by the example/dev studio here, so we shouldn't have to require users of this plugin to install it?

Reply:
that is true, but since there is no real example/dev studio (it just uses out test studio) I just thought that it might be worth adding it.

I'll remove it, I just found it very useful for when we were working since this relies so much on the right queries returning the right values n the right order. But folks can just install it again as needed without adding to the bulk

Reply:
Ah that would make sense yes haha sorry about this!